Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -44,7 +44,8 @@ public protocol ManagePledgeViewModelOutputs {
   var paymentMethodViewHidden: Signal<Bool, Never> { get }
   var pledgeDetailsSectionLabelText: Signal<String, Never> { get }
   var rightBarButtonItemHidden: Signal<Bool, Never> { get }
-  var showActionSheetMenuWithOptions: Signal<[ManagePledgeAlertAction], Never> { get }
+  /// Provides a list of `ManagePledgeAlertAction`s for the action sheet and a `Bool` indicating if the current project has  PLOT enabled
```

Comment:
There is an extra space

```suggestion
  /// Provides a list of `ManagePledgeAlertAction`s for the action sheet and a `Bool` indicating if the current project has PLOT enabled
```